src/services/software-modernisation/index.html

```diff
@@ -20,51 +20,73 @@
 
 service-line: software-modernisation
 ---
+{% assign translations = site.translations[site.lang].[page.service-line] %}
+{%- assign all-service-lines = site.translations[site.lang].all-service-lines -%}
```

Comment:
Now a bit confused whether this needs to be in service_line layout as well as here

Reply:
This assignment has been removed, however these types of assignments do need to be assigned in both the template and softmod page as there is content on both that needs it to be defined in those sections until we move the markup to the template. 

Happy to demonstrate further for clarity. 

It seems that anything pulled in through `{{ content }}` cannot pick up any assignments made in the layout, where as includes can.